docs/platforms/javascript/common/configuration/filtering.mdx

```diff
@@ -83,11 +83,116 @@ Sentry.init({
 
 You can also use `denyUrls` if you want to block specific URLs from sending exceptions to Sentry.
 
+### Using the `thirdPartyErrorFilterIntegration`
+
+_Available in browser-based SDKs from version 8.10.0_
+
+The `thirdPartyErrorFilterIntegration` allows you to filter out errors originating from third parties, such as browser extensions, code-injecting browsers, or widgets from third-party services that also use Sentry.
+This integration can be very helpful when wanting to reduce noise that is unrelated to your own application code.
+
+<Note>
+
+**Prerequisite**: To use the `thirdPartyErrorFilterIntegration`, ensure you are using a bundler and one of [Sentry's bundler plugins](https://github.com/getsentry/sentry-javascript-bundler-plugins).
+
+</Note>
+
+The integration works by "marking" your JavaScript files with an "application key" during the build process.
+At runtime, if an error occurs, the integration checks application keys for each stack frame in the stack trace.
+This allows you to filter out errors with "unmarked" stack frames, which would indicate third-party code.
+
+To set up the integration, first pass an `applicationKey` to your Sentry bundler plugin. This can be an arbitrary string that identifies your application code:
```

Comment:
```suggestion
To set up this integration, first pass an `applicationKey` to your Sentry bundler plugin. This can be an arbitrary string that identifies your application code:
```